docs/discover/search-sessions.asciidoc

```diff
@@ -59,6 +56,7 @@ image::images/search-sessions-menu.png[Search Sessions management view with acti
 
 . Use the edit menu in *Search Sessions* to:
 * *Inspect* the queries and filters that makeup the session.
+* *Edit name* of a session.
```

Comment:
Now it is also possible to edit a name of a search session: https://github.com/elastic/kibana/pull/94610